### PR TITLE
[doc] Fix Kubernetes resources documentation

### PR DESCRIPTION
**Pull request by stefan-scheidewig**
- [ ] I adjusted the corresponding [documentation](https://github.com/kobsio/kobs/tree/main/docs) for this change.
- [ ] I adjusted the [values.yaml](https://github.com/kobsio/kobs/blob/main/deploy/helm/kobs/values.yaml) file and the [documentation for all values](https://github.com/kobsio/kobs/blob/main/docs/installation/helm.md).

### CODE REVIEW

**Review by monotek — APPROVED** [review]

**Comment by stefan-scheidewig** [comment]
thanks @monotek, i can't merge this pr because i have no write access. can you please do that for me?

**Pull request merged**
